docs/firefox-logo.svg

```diff
@@ -1,3 +1,5 @@
+<svg id="logo" xmlns="http://www.w3.org/2000/svg" version="1.1" viewBox="0 0 100 100">
+<defs>
```

Comment:
Now that this is nested, the indenting looks a bit funny, but whatever... maybe I'll file a separate bug to run this through SVGO or something to see if we can shave a couple bytes from the file.

Reply:
Yeah, I can definitely fix that if it's enough of a nuisance to bother with :)